.github/workflows/create-release-tag.yaml

```diff
@@ -0,0 +1,19 @@
+name: Create release tag
+on:
+  push:
+    branches: [ develop ]
+
+jobs:
+  create-tag:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: '0'
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@1.33.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Have you checked if the default `GITHUB_TOKEN` has the rights to create a tag? It not being allowed to do some things is something that has bitten me in the past, that's why I’m asking.

Reply:
Good call out. I don't know whether this works, but also I'm not sure how to figure it out other than trying it out? (It seems safe to do so: in the worst case, it does not create a tag, and we'd have to go back to the previous, manual approach.)